src/assets/styled-components/forms.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+const styles = {
+  Form : styled.form`
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-gap: 10px;
+    background-color: purple;
+  `,
+
+  Input : styled.input`
+    background-color: #F1F1F1;
+    display: block;
+    width: 100%;
+  `,
+
+  Select : styled.select`
+    display: block;
+    width: 100%;
+  `,
+
+  Option : styled.option`
```

Comment:
I left this in purposely as boilerplate for Chase to review/use when sorting out the style guide with Styled Components.